ansible/roles/repo/tasks/redhat.yaml

```diff
@@ -124,5 +126,5 @@
     gpgcheck: true
   register: konvoy_repo_installation_rpm
   until: konvoy_repo_installation_rpm is success
-  retries: 3
-  delay: 3
```

Comment:
Was this change intentional?

Reply:
Yes, it was intentional, we are trying to keep it close to the PR that @dlipovetsky is working on. 
Should I update the commit message?